.github/workflows/errors.yml

```diff
@@ -0,0 +1,55 @@
+on: push
+
+jobs:
+  verify:
+    name: "Verify errors"
+    runs-on: ubuntu-latest
+    steps:
+      # This step checks out a copy of your repository.
+      - uses: actions/checkout@v2
+
+      # run checks on a series of directories
+      - name: Trying to run in a non-existing directory should fail
+        uses: citation-file-format/cffconvert-github-action@tests
```

Comment:
I know why use a separate branch for testing but I would move it to master branch to make it more transparent.

Reply:
I plan to replace ``@tests`` with ``@v1``, but that would currently not work since the ``tests`` version of the gh action is not published yet.